docs/protocol/network.md

```diff
@@ -0,0 +1,206 @@
+# Witnet Network Protocol
+
+Witnet network protocol is inspired in other blockchain network protocols such as Bitcoin, Ethereum and Rchain. Additionally, other blockchains such as Grin and Exonum have been taken into consideration as their reference implementations are coded in Rust.
+
+## Initial considerations
+
+In order to start the network discovery process, at least one existing node on the network is required. However, knowning several existing nodes, extracted from different sources, is always recommended in order to avoid unnecessary risks or potential attack vectors.
```

Comment:
`However, in order to avoid unnecessary risks or potential attack vectors, it is always recommended to know several existing nodes extracted from different sources.`